contentElements/InputElement.js

```diff
@@ -154,6 +154,7 @@ InputElement.prototype.isInputValid = function () {
 
 
 InputElement.prototype.toJS = function () {
+    
```

Comment:
please remove new lines that were added in this file. Your merge request should only contain files that were actually changed.

Reply:
The line was removed